openshift-knative-operator/pkg/monitoring/monitoring.go

```diff
@@ -11,6 +11,7 @@ import (
 	"github.com/openshift-knative/serverless-operator/openshift-knative-operator/pkg/common"
```

Comment:
would it make sense to have specific `monitorign` files for:
* serving
* eventing
* kafka 
?

Reply:
I dont see any difference at the moment, maybe in the future.

Reply:
I see already if lot of if/else, so perhaps make specific file for serving/eventing, with the usage of "common" util?

Adding kafka will benefit from this

Reply:
Kafka has no special handling in this context.

Reply:
This is done now